### PR TITLE
add display for peer meta

### PR DESCRIPTION
**Pull request by SWvheerden**
## Description

Adds display to peerchainmeta

## Types of changes

* [ ] Bug fix (non-breaking change which fixes an issue)
* [x] New feature (non-breaking change which adds functionality)
* [ ] Breaking change (fix or feature that would cause existing functionality to change)
* [ ] Feature refactor (No new feature or functional changes, but performance or technical debt improvements)
* [ ] New Tests
* [ ] Documentation

## Checklist:

* [x] I'm merging against the `development` branch.
* [x] I ran `cargo-fmt --all` before pushing.
* [x] I have squashed my commits into a single commit.
* [ ] My change requires a change to the documentation.
* [ ] I have updated the documentation accordingly.
* [ ] I have added tests to cover my changes.

### CODE REVIEW

File: base_layer/core/src/base_node/chain_metadata_service/handle.rs [hunk 1]
```diff
@@ -40,6 +41,13 @@ impl PeerChainMetadata {
     }
 }
 
+impl Display for PeerChainMetadata {
+    fn fmt(&self, fmt: &mut Formatter<'_>) -> Result<(), Error> {
+        fmt.write_str(&format!("Node id : {}\n", self.node_id))?;
+        fmt.write_str(&format!("Chain meta : {}\n", self.chain_metadata))
```
Comment (sdbondi) [inline base_layer/core/src/base_node/chain_metadata_service/handle.rs:47]:
```suggestion
        write!(f, "Node ID : {}\n", self.node_id))?;
        write!(f, "Chain metadata : {}\n", self.chain_metadata))
```

File: base_layer/core/src/base_node/chain_metadata_service/handle.rs [hunk 2]
```diff
@@ -40,6 +41,13 @@ impl PeerChainMetadata {
     }
 }
 
+impl Display for PeerChainMetadata {
+    fn fmt(&self, f: &mut Formatter<'_>) -> Result<(), Error> {
+        write!(f, "Node ID : {}\n", self.node_id)?;
+        write!(f, "Chain metadata : {}\n", self.chain_metadata)
```
Comment (sdbondi) [inline base_layer/core/src/base_node/chain_metadata_service/handle.rs:47]: ... I totally meant `writeln!` 😆

Comment (SWvheerden) [reply base_layer/core/src/base_node/chain_metadata_service/handle.rs:47]: Well the \n sits in a new line ...

Comment (SWvheerden) [reply base_layer/core/src/base_node/chain_metadata_service/handle.rs:47]: But it uses writeln! now...

**Pull request merged**